spec/javascript/packages/document-capture/components/review-issues-step-spec.jsx

```diff
@@ -192,9 +192,7 @@ describe('document-capture/components/review-issues-step', () => {
 
     expect(
       getByRole('link', { name: 'idv.troubleshooting.options.get_help_at_sp links.new_tab' }).href,
-    ).to.equal(
-      'https://example.com/?step=document_capture&location=document_capture_troubleshooting_options',
-    );
+    ).to.equal('https://example.com/?step=document_capture&location=post_submission_warning');
```

Comment:
this component has a default location  value of `document_capture_troubleshooting_options` but the location value passed to the component in `review_issues_step.tsx` was changed in a more recent pr to `post_submission_warning`.